os/board/rtl8721csm/src/component/os/tizenrt/rtk_blemgr.c

```diff
@@ -0,0 +1,419 @@
+			for (i = 0; i < TRBLE_BD_ADDR_MAX_LEN; i++) {
+				TRBLE_TEST_INFO("%02x ", dev->hwaddr[i]);
+			}
+			TRBLE_TEST_INFO("\n");
+		}
+	}
+	return ret;
+}
+
+trble_result_e trble_netmgr_deinit(struct bledev *dev)
+{
+	trble_result_e ret = TRBLE_FAIL;
+#ifdef CONFIG_AMEBAD_BLE_SCATTERNET
+	ret = rtw_ble_combo_deinit();
+#elif defined(CONFIG_AMEBAD_BLE_CENTRAL)
+	ret = rtw_ble_client_deinit();
+#elif defined(CONFIG_AMEBAD_BLE_PERIPHERAL)
+	ret = rtw_ble_server_deinit();
+#endif
+	if (ret == TRBLE_SUCCESS) {
+		memset(dev->hwaddr, 0, TRBLE_BD_ADDR_MAX_LEN);
+	}
+	return ret;
+}
+
+trble_result_e trble_netmgr_get_mac_addr(struct bledev *dev, uint8_t mac[TRBLE_BD_ADDR_MAX_LEN])
+{
+	memcpy(mac, dev->hwaddr, TRBLE_BD_ADDR_MAX_LEN);
+
+	TRBLE_TEST_INFO("BLE MAC : ");
+	for (int i = 0; i < TRBLE_BD_ADDR_MAX_LEN; i++) {
+		TRBLE_TEST_INFO("%02x ", mac[i]);
+	}
+	TRBLE_TEST_INFO("\n");
+	return TRBLE_SUCCESS;
+}
+
+trble_result_e trble_netmgr_disconnect(struct bledev *dev, trble_conn_handle con_handle, trble_mode_e mode)
+{
+	trble_result_e ret = TRBLE_FAIL;
+
+	if (mode == TRBLE_MODE_CLIENT) {
+		ret = rtw_ble_client_disconnect(con_handle);
+	} else if (mode == TRBLE_MODE_SERVER) {
+		ret = rtw_ble_server_disconnect(con_handle);
+	}
+	return ret;
+}
+
+trble_result_e trble_netmgr_delete_bond(struct bledev *dev, trble_bd_addr *addr, trble_mode_e mode)
+{
+	trble_result_e ret = TRBLE_FAIL;
+	if (mode == TRBLE_MODE_CLIENT) {
+		ret = rtw_ble_client_delete_bond(addr);
+	} else if (mode == TRBLE_MODE_SERVER) {
+		ret = rtw_ble_server_delete_bonded_device(addr->bd_addr);
+	}
+	return ret;
+}
+
+trble_result_e trble_netmgr_delete_bond_all(struct bledev *dev, trble_mode_e mode)
+{
+	trble_result_e ret = TRBLE_FAIL;
+	if (mode == TRBLE_MODE_CLIENT) {
+		ret = rtw_ble_client_delete_bond_all();
+	} else if (mode == TRBLE_MODE_SERVER) {
+		ret = rtw_ble_server_delete_bonded_device_all();
+	}
+	return ret;
+}
+
+/*** Central(Client) ***/
+trble_result_e trble_netmgr_start_scan(struct bledev *dev, trble_scan_filter *filter)
+{
+	trble_result_e ret = TRBLE_FAIL;
+	if (filter == NULL) {
+		ret = rtw_ble_client_start_scan();
+	} else {
+		ret = rtw_ble_client_start_scan_with_filter(filter);
+	}
+	return ret;
+}
+
+trble_result_e trble_netmgr_stop_scan(struct bledev *dev)
+{
+	trble_result_e ret = TRBLE_FAIL;
+	ret = rtw_ble_client_stop_scan();
+	return ret;
+}
+
+trble_result_e trble_netmgr_connect(struct bledev *dev, trble_bd_addr *addr)
+{
+	trble_result_e ret = TRBLE_FAIL;
+	ret = rtw_ble_client_connect(addr, addr->is_secured_connect);
+	return ret;
+}
+
+trble_result_e trble_netmgr_disconnect_all(struct bledev *dev)
+{
+	trble_result_e ret = TRBLE_FAIL;
+	ret = rtw_ble_client_disconnect_all();
+	return ret;
+}
+
+trble_result_e trble_netmgr_connected_device_list(struct bledev *dev, trble_connected_list *out_connected_list)
+{
+	trble_result_e ret = TRBLE_FAIL;
+	ret = rtw_ble_client_read_connected_device_list(out_connected_list);

```

Comment:
it's trivial how about to define ret line 254.

Reply:
Ok, i will remove 'ret'